services/.rubocop.yml

```diff
@@ -73,20 +73,20 @@ Naming/ClassAndModuleCamelCase:
 Naming/FileName:
   Enabled: true
   Exclude:
-    - 'QuillLMS/config/initializers/rack-attack.rb'
+    - "QuillLMS/config/initializers/rack-attack.rb"
 
 Naming/MethodName:
   Enabled: true
   Exclude:
-    - 'QuillLMS/app/serializers/lesson_planner/unit_serializer.rb'
+    - "QuillLMS/app/serializers/lesson_planner/unit_serializer.rb"
 
 Naming/VariableNumber:
   Enabled: true
 
 Naming/VariableName:
   Enabled: true
   Exclude:
-    - 'QuillLMS/lib/tasks/firebase.rake'
+    - "QuillLMS/lib/tasks/firebase.rake"
```

Comment:
All firebase references should have been removed in #7204
Thus, we can remove this line and also remove the file